### PR TITLE
e2e tests. Download old version of the file

### PR DESCRIPTION
**Pull request by ScharfViktor**
- added step download old version
- deleted old acceptance test (should have worked, but didn't anyway). 
- deleted line in expected failures
- closed issue: https://github.com/owncloud/ocis/issues/2261

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oCIS</strong> https://drone.owncloud.com/owncloud/web/25550/12/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/25550/tracing/folder-alice-2022-5-12-02-04-54.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/25550/tracing/folder-brian-2022-5-12-02-05-01.zip 


</p></details>

File: tests/acceptance/features/webUIFilesActionMenu/versions.feature [hunk 1]
```diff
@@ -100,12 +100,3 @@ Feature: Versions of a file
...
     Then the content of file "lorem-file.txt" for user "Alice" should be "new lorem content" in the server
     And the versions list should contain 2 entries
-
-
-  Scenario: user downloads a previous version of the file
-    Given user "Alice" has uploaded file with content "lorem" to "lorem.txt" in the server
-    And user "Alice" has uploaded file with content "lorem content" to "lorem.txt" in the server
-    And user "Alice" has logged in using the webUI
-    And the user browses to display the "versions" details of file "lorem.txt"
-    When the user downloads a previous version of the file using the webUI
```
Comment (pascalwengerter) [inline tests/acceptance/features/webUIFilesActionMenu/versions.feature:110]: Could you delete the corresponding acceptance test code as well for this step?:)

File: tests/e2e/cucumber/features/integrations/share.oc10.feature [hunk 2]
```diff
@@ -47,6 +47,9 @@ Feature: share
       | resource          | to               | create_version |
       | PARENT/simple.pdf | folder_to_shared | true           |
     #Then "Alice" should see that the resource "folder_to_shared/simple.pdf" has 1 version
+    And "Brian" downloads old version of the following resource
```
Comment (pascalwengerter) [inline tests/e2e/cucumber/features/integrations/share.oc10.feature:50]: Kind of missing a `Then` step here, or are we just trying to download and would run into a failure if it doesn't work?

**Review by pascalwengerter — COMMENTED** [review]
Probably needs an oCIS commit ID bump, otherwise two minor nitpicks

File: tests/acceptance/features/webUIFilesActionMenu/versions.feature (hunk 1 above)
Comment (ScharfViktor) [reply tests/acceptance/features/webUIFilesActionMenu/versions.feature:110]: Right, thanks for pointing that out.

File: tests/e2e/cucumber/features/integrations/share.oc10.feature (hunk 2 above)
Comment (ScharfViktor) [reply tests/e2e/cucumber/features/integrations/share.oc10.feature:50]: yes, all checks are inside steps "And" or "When".  I honestly don't know why we left the commented "Then". maybe we'll delete commented steps later

**Review by pascalwengerter — APPROVED** [review]

**Pull request merged**